apps/examples/testcase/le_tc/kernel/tc_libc_signal.c

```diff
@@ -333,6 +362,7 @@ int libc_signal_main(void)
 	tc_libc_signal_raise();
 	tc_libc_signal_sigset();
 	tc_libc_signal_signal();
+	tc_libc_signal_psignal();
```

Comment:
Please move this to the top(alphabet order.)

Reply:
modify